src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,34 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * import java.io.IOException; // Ok
+ * import java.net.URL; // Ok
+ *
+ * import java.io.IOException; // Violation, Extra separation and wrong order
+ *
+ * import javax.net.ssl.SSLContext; // Violation, Extra separation in import group
+ * import javax.net.ssl.TrustManager; // Ok
+ *
+ * import org.apache.http.conn.ClientConnectionManager; // Violation, Extra separation
+ * import org.apache.http.conn.scheme.Scheme; // Ok
+ *
+ * import java.util.Set; // Violation, Extra separation and wrong order
```

Comment:
there is already example of same vioaltion

Reply:
it makes things more clear, like for the 1st code example
should i  remove it?

Reply:
How it is different from line 155? If there is a reason, it is better to explain it in comment for violation
> could you please comment your place time in your next comment

not sure why you need this, but it is 4:56 PM

Reply:
>  it is different from line 155? 
there is no difference except the group changed 
and for the 1st example it shows many different groups separated which should not be separated.

Reply:
I still think it is not necessary